Resources/doc/cookbook/recipe_sortable_listing.rst

```diff
@@ -49,13 +49,13 @@ Then we need to inject the Sortable listener. If you only have the Gedmo bundle
 
 .. code-block:: yaml
 
-	services:
-        gedmo.listener.sortable:
+    services:
+        gedmo.listener.sortable: true
```

Comment:
Good catch! It is a mistake, I added that when trying to find what exactly was invalid